deploy/osd-machine-api/010-machine-api.MachineHealthCheck.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineHealthCheck
+metadata:
+  name: srep-healthcheck
+  namespace: openshift-machine-api
+spec:
+  selector:
+    matchLabels:
```

Comment:
Is this an OR or an AND?

Reply:
Ugh, I could have sworn that I read it was an OR but after checking the docs [0] again it's an AND. Looks like I'll need another mhm resource and split the two.

[0] https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1?tab=doc#LabelSelector